OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -105,28 +111,28 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 					var hasRequires = required.Any();
 					var isActivity = mi.HasAttribute<ScriptActorPropertyActivityAttribute>();
 
-					Console.WriteLine("<tr><td width=\"50%\" align=\"right\"><strong>{0}</strong>", mi.LuaDocString());
+					Console.Write($"| ***{mi.LuaDocString()}***");
 
 					if (isActivity)
-						Console.WriteLine("<br /><em>Queued Activity</em>");
+						Console.WriteLine("<br />*Queued Activity*");
```

Comment:
Would changing this help perhaps?
```suggestion
						Console.Write("<br />*Queued Activity*");
```

Reply:
(Because it looks like this is adding another new line, which could be an issue?)

Reply:
Fixed it.